codewof/templates/users/invitation-card.html

```diff
@@ -0,0 +1,8 @@
+<div id="invitation-{{ invitation.pk }}" class="card mb-3">
+  <div class="card-body">
+    <h5 class="card-title">{{ invitation.group.name }}</h5>
+    <h6 class="card-subtitle mb-2 text-muted">Invited by {{ invitation.inviter.full_name }}</h6>
```

Comment:
Would it be good to list the expiry date of the invitation on this card?

Reply:
Good idea. Done.